doc/api/stream.md

```diff
@@ -1499,16 +1513,24 @@ internal to the class that defines it, and should never be called directly by
 user programs.
 
 #### readable.push(chunk[, encoding])
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/11608
+    description: The `chunk` argument can now be a `Uint8Array` instance.
+-->
 
-* `chunk` {Buffer|Null|String} Chunk of data to push into the read queue
+* `chunk` {Buffer|Uint8Array|Null|String} Chunk of data to push into
```

Comment:
@seishun I agree, but I recall others possibly arguing for the uppercase version in the past? There might need to be changes to the doc tool...